clients/filter/data.go

```diff
@@ -0,0 +1,52 @@
+package filter
+
+// Dimension represents a dimension response from the filter api
+type Dimension struct {
+	Name string `json:"name"`
+	URI  string `json:"dimension_url"`
+}
+
+// DimensionOption represents a dimension option from the filter api
+type DimensionOption struct {
+	DimensionOptionsURL string `json:"dimension_option_url"`
+	Option              string `json:"option"`
+}
+
+// Model represents a model returned from the filter api
+type Model struct {
```

Comment:
Possibly need some omitempty's as the filter can be posted in bits and pieces - downloads might not be passed in initially, events might only get created one it's submitted and the system takes over